com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -388,12 +388,14 @@ static void RenderSingleCamera(ScriptableRenderContext context, CameraData camer
 
             cameraData.xr.EndCamera(cmd, cameraData);
             context.ExecuteCommandBuffer(cmd); // Sends to ScriptableRenderContext all the commands enqueued since cmd.Clear, i.e the "EndSample" command
-            CommandBufferPool.Release(cmd);
+            cmd.Clear();
 
             using (new ProfilingScope(cmd, Profiling.Pipeline.Context.submit))
```

Comment:
I think the correct fix is to pass null into ProfilingScope. 
The point is to profile context.Submit(); which doesn't take cmd, so the **scope** in cmdbuf will be empty with this modification.